### ISSUE TITLE
Add a notification when the battery is low

### ISSUE THREAD

**Issue by theodorechle** [UpsilonNumworks/Upsilon#333]
English :

Can you add a pop-up notification to warn about the low-level of the battery ?

Français :

Pouvez-vous rajouter une notification pop-up pour avertir d'un faible niveau de batterie ?

**Comment by Yaya-Cout** [comment]
Cab you share how would you like this to work ? A full screen popup like for empty battery ? A Android-style notification with a banner ? Any other way ?

**Comment by AngeDieu** [comment]
I would say like the existing banners (forbiddent app, usb connected, update available...), with an OK/EXE press to hide.

**Comment by theodorechle** [comment]
Yes or maybe more like the "no enough memory" message on the calculation app.

**Comment by lolocomotive** [comment]
> Cab you share how would you like this to work ? A full screen popup like for empty battery ? A Android-style notification with a banner ? Any other way ?

Can't wait for an android style notification system to be added to upsilon :troll:
